cmd/pd-server/main.go

```diff
@@ -37,13 +37,13 @@ import (
 
 func main() {
 	cfg := config.NewConfig()
-	err := cfg.Parse(os.Args[1:])
-
 	if cfg.Version {
 		server.PrintPDInfo()
 		exit(0)
 	}
 
+	err := cfg.Parse(os.Args[1:])
```

Comment:
I think you should parse the arguments first to fill the value of `cfg.Version`, or it will be an empty value.

Reply:
@shafreeck, got it! Thanks , please close the pr.

Reply:
> I think you should parse the arguments first to fill the value of `cfg.Version`, or it will be an empty value.

Please remove the PR, thanks!